.cspell-partial.json

```diff
@@ -1,9 +1,11 @@
 {
-  "ignorePaths": [
-    "**/perception/**",
-    "sensing/tier4_pcl_extensions/include/**",
-    "perception/bytetrack/lib/**"
-  ],
+  "ignorePaths": ["sensing/tier4_pcl_extensions/include/**", "perception/bytetrack/lib/**"],
```

Comment:
The spell check fails in tier4_pcl_extensions are fixed at https://github.com/autowarefoundation/autoware.universe/pull/6434.

Reply:
```suggestion
  "ignorePaths": ["perception/bytetrack/lib/**"],
```